charts/cluster-agent/values.yaml

```diff
@@ -69,3 +71,59 @@ clusterAgent:
   tolerations: []
 
   affinity: {}
+
+# Prometheus Config
+
+metrics:
+  # enable epsagon auto monitoring
+  enabled: false
+
+  redis:
+    # monitoring.redis.enabled -- enable redis monitoring
+    enabled: false
+
+  rabbitmq:
+    # monitoring.rabbitmq.enabled -- enable rabbitmq monitoring
+    enabled: false
+
+  mongodb:
+    # monitoring.mongodb.enabled -- enable mongodb monitoring
+    enabled: false
+
+
+metrics-agent:
+  alertmanager:
+    enabled: false
+  nodeExporter:
+    enabled: false
+  pushgateway:
+    enabled: false
+
+  serviceAccounts:
+    alertmanager:
+      create: false
+    nodeExporter:
+      create: false
+    pushgateway:
+      create: false
+
+  server:
+    remoteWrite: []
```

Comment:
How does the remote write with the user token and cluster name works?

Reply:
metrics-agent:
  server:
    remoteWrite:
      - url: https://dev.collector.epsagon.com/ingestion?d6796942-bac6-47bd-b985-db811c137d86
        basic_auth:
          username: "d6796942-bac6-47bd-b985-db811c137d86"

Reply:
but it supposed to be configured automatically for the user and not manually, and the cluster name should also be added to the remote write from the set params the user gave when he installed the chart.

Reply:
cant be configured automatically, because we cant use a value in the values file

Reply:
open to ideas

Reply:
Worst case is forking the prometheus helm chart and changing where the remote write value is used to a custom variable that we will define in the .tpl file or checking if the tpl file can override existing values. I just think that without this option we lose the whole point of making a single helm chart, if the user still needs to create a local values file and fill all the information or using a huge set commands

Reply:
i agree, ill look into forking the prometheus chart, but i think its out of the scope of this pr, merging this for now, and in the near future we'll fork the prom chart to avoid the remote write